guides/v2.1/frontend-dev-guide/layouts/layout-types.md

```diff
@@ -14,6 +14,8 @@ All page layouts used for page rendering should be declared in the page layout d
 
 Page configuration is also an .xml file. It defines the detailed structure (page header, footer, etc.), contents and page meta information, including the page layout used. Page configuration features both main elements, [blocks of particular classes]({{ page.baseurl }}/frontend-dev-guide/layouts/layout-overview.html) and containers.
 
+The page layout is specified in a layout XML file, in the root `<page>` node using the it layout argument.
```

Comment:
Hi @abrarpathan19. This information is already provided within a scope of this documentation with additional details below:

https://github.com/magento/devdocs/pull/4625/files#diff-68750291254099807104b8a2c977442aR91